test/cbmc/proofs/stringBuilderUInt32Decimal/stringBuilderUInt32Decimal_harness.c

```diff
@@ -0,0 +1,50 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Insert copyright notice
+ */
```

Comment:
Does this need to be updated to have the copyright notice? If not, delete this.

Reply:
Removed it in 327af27